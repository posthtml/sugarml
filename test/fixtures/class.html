p.foo(class='wow') bar
