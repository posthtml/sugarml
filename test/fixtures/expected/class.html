<p class="foo wow">bar</p>
